specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -464,559 +464,2343 @@
+                    "$ref": "#/definitions/ActionProperties"
+                }
+            }
+        },
+        "ActionProperties": {
+            "type": "object",
+            "description": "Action property bag.",
+            "properties": {
+                "triggerUri": {
+                    "type": "string",
+                    "description": "The uri for the action to trigger."
+                },
+                "ruleId": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "The unique identifier of the rule."
+                }
+            }
+        },
+        "CaseList": {
+            "description": "List all the cases.",
+            "properties": {
+                "nextLink": {
+                    "readOnly": true,
+                    "description": "URL to fetch the next set of cases.",
+                    "type": "string"
+                },
+                "value": {
+                    "description": "Array of cases.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Case"
+                    }
+                }
+            },
+            "required": [
+                "value"
+            ]
+        },
+        "Case": {
+            "type": "object",
+            "description": "Represents a case in Azure Security Insights.",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "properties": {
+                "etag": {
+                    "type": "string",
+                    "description": "Etag of the alert rule."
+                },
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Case data",
+                    "$ref": "#/definitions/CaseProperties"
+                }
+            }
+        },
+        "CaseProperties": {
+            "type": "object",
+            "description": "Describes case properties",
+            "properties": {
+                "lastUpdatedTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The last time the case was updated"
+                },
+                "createdTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The time the case was created"
+                },
+                "endTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The end time of the case"
+                },
+                "startTimeUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The start time of the case"
+                },
+                "tags": {
```

Comment:
everywhere, tags should be a top level property. Is it modeled like this since tags on proxy resource is not supported?

Reply:
It will have a bit different behavior than regular tags, and in order to avoid the confusion we will change it to labels